miner/README.org

```diff
@@ -25,24 +25,24 @@
 
 * What is Mining?
 
-A blockchain is a series of /blocks/. Blocks contain /transactions/ which
+A blockchain is a series of /blocks/. Blocks contain /transactions/ that
 represent coin transfers or other Smart Contract interactions. Mining is the
 process of mathematically "solving" a block. Unless solved, a block cannot be
 included in the chain. Without mining, a blockchain cannot progress and
 transactions will never finalize.
 
-Economically, mining is the principal way to obtain currency on a Chainweb
-network. Since Chainweb uses Proof-of-Work, computational effort is rewarded
+Economically, mining is the principal way to obtain currency on a Kadena
+network. Since Kadena uses Proof-of-Work, computational effort is rewarded
```

Comment:
Is the Chainweb name going away? It's unclear in this line if "Kadena" refers to the network, the protocol, the company, etc.

Reply:
You are correct. Moving forward, we will no longer be using Chainweb for user/client-facing documents. We can use chainweb, but only when speaking about chainwebs in general. 

For line 34, how would you position that? 'Kadena public blockchain network'?